### ISSUE TITLE
replaceWithText should not replace jsdocs

### ISSUE THREAD

**Issue by dsherret** [dsherret/ts-simple-ast#417]
I'm thinking that `replaceWithText` should probably do `.getStart()` instead of `.getStart(true)`. The reason for this is because `.getText()` returns `.getStart()` to `.getEnd()` and there is no easy way to get the text from `.getStart(true)`.

**Comment by ihorskyi** [comment]
BTW, is there a way to add/replace jsdocs/comments using `ts-simple-ast`?

**Comment by dsherret** [comment]
@ihorskyi yeah, nodes that can have jsdocs like properties, classes, etc. will have methods for adding or inserting them (ex. `addJsDoc(...)`, `insertJsDoc(...)`). To replace the comment within the jsdoc, use `jsDoc.setComment(...)`.

**Comment by Jack-Works** [comment]
Hi, is there a way to replace JSDoc? I'm handling JSDoc typed JS code and try to replace them with TypeScript flavor JSDoc comment so TypeScript can generate type for those code. I find I have to call replaceText to do this.
